modules/deploy/partials/requirements.adoc

```diff
@@ -78,22 +78,44 @@ endif::[]
 
 *Requirements*:
 
-- Two physical, not virtual, cores for each node.
-
-- x86_64 (Westmere or newer) and AWS Graviton family processors are supported.
-
-- 2 GB or more of memory per core.
+- Each node must have at least two physical CPU cores.
+- x86_64 (Westmere or newer) and AWS Graviton processors are supported.
+ifdef::env-kubernetes[]
+- Each Redpanda Pod requires at least 2 GiB of memory per core.
++
+** Request a minimum of 2.22 GiB per core to meet Redpanda's memory allocation strategy.
++
+See xref:manage:kubernetes/k-manage-resources.adoc[] for detailed guidance and examples.
+endif::[]
 
-- 2 MB of memory for each topic partition replica.
+- Each Redpanda broker must have at least 2 GB of memory per core.
 
+- Each Redpanda broker must have at least 2 MB of memory for each topic partition replica.
++
 The total memory available for partition replicas is determined as a percentage of the cluster's total memory, which is controlled by the xref:reference:tunable-properties.adoc#topic_partitions_memory_allocation_percent[`topic_partitions_memory_allocation_percent`] setting. Each partition replica consumes xref:reference:tunable-properties.adoc#topic_memory_per_partition[`topic_memory_per_partition`] bytes from this pool. If insufficient memory is available, topic operations will fail. You can adjust the allocation ratio using topic_partitions_memory_allocation_percent, but doing so is not recommended, as lowering it may lead to instability or degraded performance.
```

Comment:
```suggestion
The total memory available for partition replicas is determined as a percentage of the cluster's total memory, which is controlled by the xref:reference:tunable-properties.adoc#topic_partitions_memory_allocation_percent[`topic_partitions_memory_allocation_percent`] setting. Each partition replica consumes xref:reference:tunable-properties.adoc#topic_memory_per_partition[`topic_memory_per_partition`] bytes from this pool. If insufficient memory is available, topic operations will fail. You can adjust the allocation ratio using `topic_partitions_memory_allocation_percent`, but doing so is not recommended, as lowering it may lead to instability or degraded performance.
```
Not part of the diff, but consider the code fence for consistency!